### PR TITLE
Using correct restApi to fetch location names to create and fetch web…

### PR DESCRIPTION
**Pull request by anaggar**
…site name + adding helpmarkdown to state website listed/created will be in default webbspace

### CODE REVIEW

File: Tasks/AzureWebPowerShellDeployment/task.json [hunk 1]
```diff
@@ -29,22 +29,23 @@
...
-      }
-    },
-    {
-      "name": "WebSiteLocation",
-      "type": "pickList",
-      "label": "Web App Location",
-      "defaultValue": "South Central US",
-      "required": true,
-      "helpMarkDown": "Select a region for deployment.Possible options are **East US**, **East US 2**, **Central US**, **South Central US**, **West US**, **North Europe**, **West Europe** and others.",     
+      "helpMarkDown": "",
```
Comment (kmkumaran) [inline Tasks/AzureWebPowerShellDeployment/task.json]: Why is this empty?

File: Tasks/AzureWebPowerShellDeployment/task.loc.json [hunk 2]
```diff
@@ -29,22 +29,23 @@
       "helpMarkDown": "ms-resource:loc.input.help.ConnectedServiceName"
     },
     {
-      "name": "WebSiteName",
+      "name": "WebSiteLocation",
```
Comment (kmkumaran) [inline Tasks/AzureWebPowerShellDeployment/task.loc.json]: Are there any L0 tests that should be updated?

Comment (anaggar) [inline Tasks/AzureWebPowerShellDeployment/task.loc.json]: waiting on roopesh for message

File: Tasks/AzureWebPowerShellDeployment/task.json (hunk 1 above)
Comment (anaggar) [inline Tasks/AzureWebPowerShellDeployment/task.json]: waiting on @RoopeshNair  for message

File: Tasks/AzureWebPowerShellDeployment/task.loc.json (hunk 2 above)
Comment (anaggar) [inline Tasks/AzureWebPowerShellDeployment/task.loc.json]: not required, and test are passing

**Pull request merged**